src/utils/database.ts

```diff
@@ -0,0 +1,41 @@
+import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
+
+export class Database {
+  defaultFilePath: string;
+
+  constructor() {
+    this.defaultFilePath = 'saves/data.json';
+  }
+
+  hasGameFile(filePathOverride?: string): any {
```

Comment:
You rarely want to return `any` on purpose. This should return`boolean`.

Reply:
I added [@typescript-eslint/no-explicit-any](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-explicit-any.md#disallow-usage-of-the-any-type-no-explicit-any) to the next ticket of eslint enhancements

Reply:
Updated to return `boolean`